client/src/pages/Messaging.jsx

```diff
@@ -0,0 +1,275 @@
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={person3}
+                name="George"
+                date="1-14"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+              <Chats
+                avi={avatar}
+                name="John"
+                date="Yesterday"
+                message="Hey David, are you still available for our class at 2pm tomorrow?"
+                setOpenMsg={setOpenMsg}
+              />
+            </div>
+          ) : (
+            <div className="w-full">
+              <div className="fixed left-0 right-0 z-10 bg-white max-w-full bg-primary">
```

Comment:
bg-white and bg-primary are competing for the property so we can delete one

Reply:
Yeah, made a mistake there